src/learn_to_pick/__init__.py

```diff
@@ -21,6 +21,10 @@
     PickBestSelected,
 )
 
+from learn_to_pick.byom.pytorch_policy import PyTorchPolicy
```

Comment:
should we refactor into two directories here 'vowpal_wabbit` and `pytorch`? The policies are specific so not sure that `byom` as a name makes sense in these implementations